app/views/api/release_notes/release_notes.yml

```diff
@@ -20,7 +20,61 @@
 #     [links](https://www.gov.uk)
 #
 #     **bold** and _italic_ text
+- title: Change schedules endpoint released to the Register ECTs sandbox with updated validation rules
+  date: 2025-12-12
+  tags:
+    - sandbox-release
+  body: |
+    The `PUT /participants/{id}/change-schedule` endpoint is now available in the Register ECTs sandbox. We have also updated the validation rules for when lead providers can change a participant’s schedule or cohort.
+
+    These changes are defined in sections 19.1, 19.2 and 19.3 of the API specification.
+
+    ## What’s changed from the existing ‘Manage training for early career teachers’ API?
+
+    ### Only the most recent lead provider can update a participant’s schedule or cohort
```

Comment:
I know it feels like it's shortening it but it's also that it needs to be the CURRENT lead provider as specified in the spec.

For example, if Lead Provider 1 is training Bob, and then Lead Provider 2 is taking over on the 13th December, neither can change the schedule until Bob starts with LP2 on the 13th December.